src/pages/FormBuilder/FormBuilder.module.css

```
@@ -261,77 +221,18 @@
 }
 
 .contentCard {
+  font-family: 'Inter';
```

Stylelint (17.9.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)